lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_delete_group_action.rb

```diff
@@ -22,7 +22,16 @@ def self.run(params)
 
         UI.message("⏳ Deleting tester group '#{group_alias}' in project #{project_number}...")
 
-        client.delete_project_group(group_name(project_number, group_alias))
+        begin
+          client.delete_project_group(group_name(project_number, group_alias))
+        rescue Google::Apis::Error => err
+          case err.status_code.to_i
+          when 404
```

Comment:
Are you sure this is a 404 and not a 400?

Reply:
Yep just double checked the backend code.